bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonBridgeHandler.java

```diff
@@ -314,8 +319,9 @@ public void reconnect(InsteonModem modem) {
                 return;
             }
 
-            cancelJob(reconnectJob, false);
             updateStatus();
+
+            cancelJob(reconnectJob, false);
```

Comment:
I am not to understand why you cancel the job in its execution ?

Same remark for connectJob.

Reply:
I believed I finally understood. You have a repeated job but you want to stop it as soon as reconnect succeeded

Reply:
Correct. Both are `scheduleWithFixedDelay` tasks that should be terminated once the connection is established.